massgov/mayflower
# Patternlab/DP-10274  feedback button obscures navigation

## Description
Hide feedback button when sticky menus are open

## Related Issue / Ticket

- [JIRA issue DP-10274](https://jira.mass.gov/browse/DP-10274)

## Steps to Test


1. From any page, open the main navigations and check that the button hides
2. From an info details page with a sticky TOC, open the toc and see that the button hides

The button hides when the new org sub nav opens, that change was made with DP-9337

## Screenshots
<img width="454" alt="screen shot 2018-09-11 at 3 37 46 pm" src="https://user-images.githubusercontent.com/18662734/45386320-b2ac4d80-b5d8-11e8-9e27-e42179a4eecb.png">
<img width="453" alt="screen shot 2018-09-11 at 3 37 58 pm" src="https://user-images.githubusercontent.com/18662734/45386322-b4761100-b5d8-11e8-863d-1a7d8bf26862.png">
